pkg/scheduler/api/node_info.go

```diff
@@ -52,6 +52,8 @@ type NodeInfo struct {
 	// Used to store custom information
 	Others     map[string]interface{}
 	GPUDevices map[int]*GPUDevice
+
+	bindingTasks map[TaskID]string
```

Comment:
It's better to have a `Binding` resources, similar to `Releasing` above :)

Reply:
Any feedback on this comment?

Reply:
/cc @zen-xu

Reply:
As it's hard to judge the task state of allocating failed, `Releasing` resource is hard to implement by now.